Sources/Commands/SwiftPackageTool.swift

```diff
@@ -68,11 +68,22 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
 
         case .update:
             let workspace = try getActiveWorkspace()
-            try workspace.updateDependencies(
-                root: getWorkspaceRoot(),
-                diagnostics: diagnostics
-            )
-
+            if options.dryRun {
+                let result = try workspace.dryRun(
+                    root: getWorkspaceRoot(),
+                    diagnostics: diagnostics
+                )
+                print("\n")
+                for (package, state) in result {
+                    print(package, " ", state)
+                }
```

Comment:
Maybe we can output a formatted table, which looks something like:

| Dependency | Current | Update |
|---|---|---|
| Vapor | 1.0.1  | 1.5.0  |
| SwiftyJSON  | 9564d9  | d45d15  |
| Curl  | up-to-date  | up-to-date  |

We can also just omit the dependencies which are unchanged.

Reply:
done